packages/core/src/ColorPicker/Fields/Fields.tsx

```diff
@@ -88,12 +88,16 @@ export const Fields = (props: FieldsProps) => {
         },
         event,
       );
-    } else if (data.r || data.g || data.b) {
+    } else if (
+      data.r !== undefined ||
+      data.g !== undefined ||
+      data.b !== undefined
+    ) {
       onChange(
         {
-          r: data.r || rgb?.r,
-          g: data.g || rgb?.g,
-          b: data.b || rgb?.b,
+          r: data.r !== undefined ? data.r : rgb?.r,
+          g: data.g !== undefined ? data.g : rgb?.g,
+          b: data.b !== undefined ? data.b : rgb?.b,
```

Comment:
we have syntax for this case
```suggestion
          r: data.r ?? rgb?.r,
          g: data.g ?? rgb?.g,
          b: data.b ?? rgb?.b,
```

Reply:
Of course. Updated.